site/pages/references.html

```diff
@@ -0,0 +1,99 @@
+{% extends "base.html" %}
+{% block title %} 404 Not Found {% endblock %}
+{% block meta_description %} 404 Not Found {% endblock %}
+{% block content %}
+
+<div id="references" class="width-960">
+  <div id="hero" class="dark short">
+    <div id="hero-content" class="wide">
+      <h1 id="logo-title">
+        Hail-Powered Science
+      </h1>
+      <div class="logo-subtitle">
+        An incomplete list of scientific work enabled by Hail.
+      </div>
+    </div>
+  </div>
+  <div class="about">
+    <div class="content">
+      <p>If you use Hail for published work, please cite the software. You can get a citation for the version of Hail you installed by executing:</p>
+      <pre><code class="language-python">import hail as hl
+print(hl.citation())</code></pre>
+      <p>Or you could include the following line in your bibliography:</p>
+      <pre><code>Hail Team. Hail 0.2. https://github.com/hail-is/hail</code></pre>
+      <p>Otherwise, we welcome you to add additional examples by <a href="https://github.com/hail-is/hail/edit/main/site/pages/references.html">editing this page directly</a>, after which we will review the pull request to confirm the addition is valid. Please adhere to the existing formatting conventions.</p>
+      <p><em>Last updated on March 30th, 2020</em></p>
```

Comment:
I know you didn't write this, but this can't be right, since some of these papers are after March 30th 2020